### ISSUE TITLE
Skip authorization form when performing OAuth

### ISSUE THREAD

**Issue by algomaster99** [IMGIITRoorkee/omniport-backend#112]
**Is your feature request related to a problem? Please describe.**
It will be nice to have a feature to skip the same authorization form when performing OAuth with Omniport for the nth time.

**Describe the solution you'd like**
An option for the client in the developer service whether to enforce this authorization form every time their user tries to log in or not.

**Describe alternatives you've considered**
Follow the documentation [here](https://django-oauth-toolkit.readthedocs.io/en/latest/advanced_topics.html#skip-authorization-form).

**Additional context**
![Screenshot from 2020-07-11 19-51-03](https://user-images.githubusercontent.com/35191225/87226227-0001cd80-c3b0-11ea-9d85-124c70d9897b.png)

**Want to take up?**
:+1:

**Comment by pradumangoyal** [comment]
It's a good feature to be present, let it be there as a good-first-issue. Coming Project Associates can have their hand on such issues, before starting their dev journey with omniport.

**Comment by algomaster99** [comment]
Sounds good to me.

**Comment by dhruvkb** [comment]
Is "Project Associates" a new role nomenclature?

**Comment by algomaster99** [comment]
@dhruvkb yes! :smile:

**Comment by pradumangoyal** [comment]
@dhruvkb remeber the times we were discussing something better than hub coordinator! Also, do you remember your suggestion? 😛

**Comment by dhruvkb** [comment]
Are “webmasters” still called the same? Or did you rename their title too?

**Comment by algomaster99** [comment]
Not yet. We aren't planning but do you have a suggestion?

**Comment by dhruvkb** [comment]
> Also, do you remember your suggestion? 😛
While "webmasters" is a good enough name, ^this one works too.

**Comment by pradumangoyal** [comment]
There can be three options to select from instead of having checkboxes:
_Skip Authorisation:_
- Complete: For the Apps where the Allow from User os not at all required and will skip the form completely. Can be used for apps by the maintainer itself.
- Partial: One-time authorisation.
- Strict (Default): Always authorise

**Comment by algomaster99** [comment]
This makes sense but does `django-oauth-toolkit` have support for enabling all three options?

**Comment by pradumangoyal** [comment]
One can override the Application model, and the implementation of the options are to be done manually on the frontend view.

**Comment by MihirSachdeva** [comment]
I would like to work on this :smile:

**Comment by ktg-123** [comment]
I would like to work on this issue.. if possible

**Comment by algomaster99** [comment]
@ktg-123 you can work on this. :+1: 
@MihirSachdeva assigning you https://github.com/IMGIITRoorkee/omniport-frontend-maintainer-site/issues/25.
